client/src/pages/automation/project/components/DataPill.tsx

```diff
@@ -116,8 +116,10 @@ const DataPill = ({
                                     workflowNodeName={workflowNodeName}
                                 />
 
-                                {typeof value !== 'object' && (
-                                    <div className="flex-1 text-xs text-muted-foreground">{value}</div>
+                                {(value || value === 0 || value === false) && typeof value !== 'object' && (
+                                    <div className="flex-1 text-xs text-muted-foreground">
+                                        {value === true ? 'true' : value === false ? false : value}
```

Comment:
ovo mi izgleda kao komplicirana verzija onog šta je moglo biti samo `{value}`

1. ako `value === true` onda ispiši `true`
2. ako `value === false` onda ispiši `false` ili `value` (koji je `false`)

samo `{value}` će ispisati vrijednost od value u DOM (razlog zašto se triba forsirat boolean konverzija kod `.length` provjere)